pyinfraformat/core/io.py

```diff
@@ -133,7 +148,19 @@ def parse_line(line):
 
         if ("properties" in line) and ("ALKUPERAINEN_DATA" in line["properties"]):
             hole_str = line["properties"]["ALKUPERAINEN_DATA"].replace("\\r", "\n").splitlines()
-            hole = parse_hole(enumerate(hole_str), robust=robust)
+            if errors == "ignore_holes":
+                try:
+                    hole, illegal_rows = parse_hole(enumerate(hole_str), robust=False)
+                except ValueError:
+                    if save_ignored:
+                        collected_illegals.extend(hole_str)
+                    return False
+            elif errors == "ignore_lines":
+                hole, illegal_rows = parse_hole(enumerate(hole_str), robust=True)
+                if illegal_rows and save_ignored:
+                    collected_illegals.extend(illegal_rows)
```

Comment:
Should these contain location where they were found (e.g. linenumber?) or is that irrelevant?

Reply:
Fixed